core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -28,6 +29,20 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, csscla
                 f.entry (title:_("TCP port for JNLP agents"), field:"slaveAgentPort") {
                     f.serverTcpPort()
                 }
+                f.advanced(title: _("Agent protocols"), align:"left") {
+                    f.entry(title: _("Agent protocols")) {
+                        def agentProtocols = my.agentProtocols;
+                        for (AgentProtocol p: AgentProtocol.all()) {
+                            if (p.name != null && !p.required) {
+                                f.checkbox(name: "agentProtocol",
+                                        title: p.name,
```

Comment:
Should agent protocols have an optional display name / help file for this UI to enable users to make an actual informed decision.